yaptide/utils/sim_utils.py

```diff
@@ -33,19 +36,90 @@ def pymchelper_output_to_json(estimators_dict: dict, dir_path: Path) -> dict:
     return result_dict
 
 
-def write_input_files(json_data: dict, output_dir: Path):
+class JSON_TYPE(Enum):
+    """Class defining custom JSON types"""
+
+    Editor = auto()
+    Files = auto()
+
+
+def get_json_type(payload_dict: dict) -> JSON_TYPE:
+    """Returns type of provided JSON"""
+    possible_input_file_names = set(['beam.dat', 'geo.dat', 'detect.dat', 'mat.dat'])  # skipcq: PTC-W0018
+    if possible_input_file_names.intersection(set(payload_dict["sim_data"].keys())):
+        return JSON_TYPE.Files
+    return JSON_TYPE.Editor
+
+
+def convert_editor_payload_to_dict(editor_dict: dict, parser_type: str) -> dict:
```

Comment:
Shouldn't this be renamed to `convert_editor_dict_to_files_dict` ?
Using Visual Studio code and pressing F2 should do the job

Reply:
Done